code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -831,4 +831,51 @@
 /datum/reagent/greenvomit/reaction_turf(turf/T, volume)
 	if(volume >= 5 && !istype(T, /turf/space))
 		new /obj/effect/decal/cleanable/vomit/green(T)
-		playsound(T, 'sound/effects/splat.ogg', 50, 1, -3)
\ No newline at end of file
+		playsound(T, 'sound/effects/splat.ogg', 50, 1, -3)
+		
+////Lavaland Flora Reagents////
+
+/datum/reagent/consumable/entpoly
+	name = "Entropic Polypnium"
+	id = "entpoly"
+	description = "An ichor, derived from a certain mushroom, makes for a bad time."
+	color = "#1d043d"
+
+/datum/reagent/consumable/entpoly/on_mob_life(mob/living/M)
+	if(current_cycle >= 10)
+		M.Paralyse(2, 0)
+		. = 1
+	if(prob(20))
+		M.LoseBreath(4)
+		M.adjustBrainLoss(2 * REAGENTS_EFFECT_MULTIPLIER)
+		M.adjustToxLoss(3 * REAGENTS_EFFECT_MULTIPLIER, 0)
+		M.adjustStaminaLoss(10 * REAGENTS_EFFECT_MULTIPLIER, 0)
+		M.EyeBlurry(5)
+		. = TRUE
```

Comment:
kill this, also all the above damage procs should only have a singular argument.